vscode-dotnet-sdk-extension/src/extension.ts

```diff
@@ -0,0 +1,167 @@
+
+            let version: string | undefined = commandContext ? commandContext.version : undefined;
+            if (!version) {
+                version = await vscode.window.showInputBox({
+                    placeHolder: '5.0',
+                    value: '5.0',
+                    prompt: '.NET version, i.e. 5.0',
+                });
+            }
+            if (!version) {
+                displayWorker.showErrorMessage('No .NET SDK version provided', () => { /* No callback needed */ });
+                return undefined;
+            }
+
+            eventStream.post(new DotnetAcquisitionRequested(version!));
+            const resolvedVersion = await versionResolver.getFullSDKVersion(version!);
+            const dotnetPath = await acquisitionWorker.acquireSDK(resolvedVersion);
+            displayWorker.showInformationMessage(`.NET SDK ${version} installed to ${dotnetPath.dotnetPath}`, () => { /* No callback needed */ });
+            setPathEnvVar(path.dirname(dotnetPath.dotnetPath));
+            return dotnetPath;
+        }, issueContext(undefined, 'acquireSDK'));
+        return pathResult;
+    });
+    const dotnetUninstallAllRegistration = vscode.commands.registerCommand(`${commandPrefix}.${commandKeys.uninstallAll}`, async (commandContext: IDotnetUninstallContext | undefined) => {
+        await callWithErrorHandling(async () => {
+            await acquisitionWorker.uninstallAll();
+            displayWorker.showInformationMessage('All VS Code copies of the .NET SDK uninstalled.', () => { /* No callback needed */ });
+        }, issueContext(commandContext ? commandContext.errorConfiguration : undefined, 'uninstallAll'));
+    });
+    const showOutputChannelRegistration = vscode.commands.registerCommand(`${commandPrefix}.${commandKeys.showAcquisitionLog}`, () => outputChannel.show(/* preserveFocus */ false));
+    const reportIssueRegistration = vscode.commands.registerCommand(`${commandPrefix}.${commandKeys.reportIssue}`, async () => {
+        const [url, issueBody] = formatIssueUrl(undefined, issueContext(AcquireErrorConfiguration.DisableErrorPopups, 'reportIssue'));
+        await vscode.env.clipboard.writeText(issueBody);
+        open(url);
+    });
+
+    context.subscriptions.push(
+        dotnetAcquireRegistration,
+        dotnetUninstallAllRegistration,
+        showOutputChannelRegistration,
+        reportIssueRegistration);
+    context.subscriptions.push({
+        dispose: () => vscode.Disposable.from(...eventStreamObservers).dispose(),
+    });
+}
+
+function setPathEnvVar(pathAddition: string) {
+    let pathCommand: string;
+    if (os.platform() === 'win32') {
+        pathCommand = `setx PATH "${pathAddition};%PATH%"`;
+    } else {
+        pathCommand = `export PATH="${pathAddition};$PATH"`;
+    }
+    try{
+        cp.execSync(pathCommand);
+    } catch (error) {
+        throw new Error(`Unable to add SDK to the PATH: ${error}`);
```

Comment:
Because of how this is called wont this result in us prompting the user to file an issue? I feel like we should be utilizing a VSCode prompt here to suggest a help article or at the very least provide the path to the user so they can fully qualify their dotnet invocations. Kathleen probably has better opinions here than I though 😄